roles/network_plugin/calico/tasks/main.yml

```diff
@@ -190,5 +185,8 @@
     - {name: calico, file: calico-crb.yml, type: clusterrolebinding}
   register: calico_node_manifests
   when:
-    - inventory_hostname in groups['kube-master']
-    - rbac_enabled or item.type not in rbac_resources
+    - calico_upgrade_needed
```

Comment:
I can't find the task that sets calico_upgrade_needed to true

Reply:
pushed missing file